solver/src/main/java/org/chocosolver/solver/constraints/nary/binPacking/PropBinPacking.java

```diff
@@ -348,6 +336,16 @@ public void propagate(int idxVarInProp, int mask) throws ContradictionException
     @Override
     public void propagate(int evtmask) throws ContradictionException {
         if(PropagatorEventType.isFullPropagation(evtmask)) {
+            for(int j = 0; j<nbAvailableBins; j++) {
```

Comment:
to be 100% safe, P[j] and R[j] should be cleared here I think

Reply:
I agree, reifying this constraint could lead to unexpected results if `P[j]` and `R[j]` are not cleared.

Reply:
Yes, I agree it would be safer (and it is now done)